src/storage/config.rs

```diff
@@ -43,6 +48,8 @@ pub struct Config {
     pub scheduler_pending_write_threshold: ReadableSize,
     #[config(skip)]
     // Reserve disk space to make tikv would have enough space to compact when disk is full.
+    // Values less than `5GB` will be sanitized to `5GB` (2GB for `max_compaction_bytes`, and
```

Comment:
What if `max_compaction_bytes` is altered?

Reply:
Seems no better solutions except explain it in config file?

Reply:
Yes, 5G should be safe enough.